rebar.config.script

```diff
@@ -32,7 +32,15 @@ CouchJSName = case os:type() of
         "couchjs"
 end,
 CouchJSPath = filename:join(["priv", CouchJSName]),
-Version = string:strip(os:cmd("git describe --always"), right, $\n),
+Version0 = string:strip(os:cmd("git describe --always"), right, $\n),
+% When we are not building from a git checkout, the above will fail.
+% If that is the case, we use the COUCHDB_VERSION variable that has
+% been set by the top level Makefile
+Version = case Version0 of
+    "fatal:" ++ _Msg ->
```

Comment:
If there are no git on the target system, this message will be different:
```
bash: git: command not found
```

Other shells may give different error messages. May be use another strategy here:
- If COUCHDB_VERSION is set - use it
- Else try call git describe
- If success (exit status 0) - use that value
- Otherwise set version as "unknown" and cry loudly